src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js

```diff
@@ -845,6 +850,13 @@ class CJ4_FMC extends FMCMainDisplay {
                 break;
         }
     }
+
+    updateMsgs() {
```

Comment:
Can we make this delta time based instead of a frame counter?

Reply:
yeah we should replace all those in cj4_fmc with time based values

Reply:
as we should resolve this in general, this should go to a different pr